backend/users/models.py

```diff
@@ -1,3 +1,34 @@
+import os
+import uuid
+
+from django.contrib.auth.models import AbstractUser
 from django.db import models
 
-# Create your models here.
+
+def get_file_path(instance, filename):
+    filename = f"avatar_image_{instance.user.id}.jpeg"
+
+    return os.path.join("images/", filename)
```

Comment:
Is it really necessary to create a variable when you use it only once?
```suggestion
    return os.path.join("images/", f"avatar_image_{instance.user.id}.jpeg")
```